### ISSUE TITLE
caretRangeFromPoint null

### ISSUE THREAD

**Issue by 79144876** [naoak/WebViewMarker#8]
if i set my device language to France or other. 
```
 var range = document.caretRangeFromPoint(android.selection.lastTouchPoint.x, android.selection.lastTouchPoint.y);
```
range will be return null ,what's wrong?
It  only work on english device success.
Thanks.

**Comment by 79144876** [comment]
But https://github.com/btate/BTAndroidWebViewSelection  work success,
What's the different ?

**Comment by 79144876** [comment]
got it
```
 String startTouchUrl = String.format(Locale.US, "javascript:android.selection.startTouch(%f, %f);",
                    xPoint, yPoint);
```

**Comment by naoak** [comment]
LTNS.
This issue was fixed by #6.
Will be work on v0.1.3.

**Comment by naoak** [comment]
If you still have any problems, reopen this.
